src/main/java/com/marginallyclever/makelangelo/makeArt/imageConverter/Converter_CMYK.java

```diff
@@ -34,9 +34,15 @@ public int getPasses() {
 		return passes;
 	}
 	
+	/**
+	 * Passing a value lower than 1 set passes value to 1.
+	 * @param value 
+	 */
 	public void setPasses(int value) {
-		if(passes<1) passes=1;
-		passes=value;
+		if(value<1) 
+		    passes=1;
+		else 
+		    passes=value;
```

Comment:
passes=(int)Math.max(1,value)?  your way is good, too.  thank you for adding javadoc.